hugo/layouts/partials/distrib_map.html

```diff
@@ -35,20 +35,12 @@ <h4 class="me-auto">Help</h4>
 </div>
 
 
-<link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css"
-    integrity="sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY=" crossorigin="" />
-
-<script src="https://unpkg.com/leaflet@1.9.4/dist/leaflet.js"
-    integrity="sha256-20nQCchB9co0qIjJZRGuk2/Z9VM+kNiyxNV1lvTlZBo=" crossorigin=""></script>
-
-
+<!-- Leaflet observation map script -->
 {{ $latitude := .Params.latitude | default 0 }}
 {{ $longitude := .Params.longitude | default 0 }}
 {{ $initialZoom := .Params.initialZoom | default 1 }}
 
-
 <script>
-
     const map = L.map('map', {
```

Comment:
To speed up page load, have you considered adding the `defer` attribute to the leaflet script, and wrapping the map code in an event listener `document.addEventListener("DOMContentLoaded", function() { /* Map code here */ }` ? I'm not sure about this but wanted to evoke the possibility.

Reply:
I see what you mean and think it could help. I'm not sure it matters but it is important perhaps that the map is "Above the fold content", but I can investigate it (Jira EB-198)! :)